### PR TITLE
Aladdin generated examples for KeyVault

### PR DESCRIPTION
**Pull request by glGarg**
- Sort commands alphabetically in keyvault _help.py file for easy diff.
- Add autogenerated examples.
---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

File: src/command_modules/azure-cli-keyvault/azure/cli/command_modules/keyvault/_help.py [hunk 1]
```diff
@@ -28,7 +28,7 @@
 long-summary: Certificates can be used as a secrets for provisioned virtual machines.
 examples:
   - name: Create a self-signed certificate with the default policy and add it to a virtual machine.
-    text: |
+    text: |-
```
Comment (tjprescott) [inline src/command_modules/azure-cli-keyvault/azure/cli/command_modules/keyvault/_help.py]: What is the `-` addition for?

File: src/command_modules/azure-cli-keyvault/azure/cli/command_modules/keyvault/_help.py [hunk 2]
```diff
@@ -39,6 +39,9 @@
 
         az vm create -g group-name -n vm-name --admin-username deploy  \\
           --image debian --secrets "$vm_secrets"
+  - name: Create a Key Vault certificate. (autogenerated)
+    text: "az keyvault certificate create --name cert1 \\\n  --policy \"$(az keyvault certificate get-default-policy)\" --vault-name vaultname"
```
Comment (tjprescott) [inline src/command_modules/azure-cli-keyvault/azure/cli/command_modules/keyvault/_help.py]: What's up with the `\\\n`?

File: src/command_modules/azure-cli-keyvault/azure/cli/command_modules/keyvault/_help.py [hunk 3]
```diff
@@ -93,6 +99,9 @@
 
         az vm create -g group-name -n vm-name --admin-username deploy  \\
           --image debian --secrets "$vm_secrets"
+  - name: Import a certificate into KeyVault. (autogenerated)
+    text: "az keyvault certificate import --file cert_file\n\nsecrets=$( --name vaultname --vault-name vaultname"
```
Comment (tjprescott) [inline src/command_modules/azure-cli-keyvault/azure/cli/command_modules/keyvault/_help.py]: Screwed up syntax.

File: src/command_modules/azure-cli-keyvault/azure/cli/command_modules/keyvault/_help.py (hunk 1 above)
Comment (glGarg) [reply src/command_modules/azure-cli-keyvault/azure/cli/command_modules/keyvault/_help.py]: That is valid yaml syntax but I'll get rid of it

File: src/command_modules/azure-cli-keyvault/azure/cli/command_modules/keyvault/_help.py (hunk 2 above)
Comment (glGarg) [reply src/command_modules/azure-cli-keyvault/azure/cli/command_modules/keyvault/_help.py]: Those are escape and newline characters for multiline examples

Comment (tjprescott) [reply src/command_modules/azure-cli-keyvault/azure/cli/command_modules/keyvault/_help.py]: They shouldn't be present in the text.

File: src/command_modules/azure-cli-keyvault/azure/cli/command_modules/keyvault/_help.py (hunk 3 above)
Comment (glGarg) [reply src/command_modules/azure-cli-keyvault/azure/cli/command_modules/keyvault/_help.py]: Yaml dumper we use just decides to add these on it's own. Fixing this.

**Comment by tjprescott** [comment]
Closing pending resolution of code-generation issues.

**Pull request closed without merging**